addons/escoria-core/plugin.gd

```diff
@@ -37,85 +37,86 @@ func _ready():
 
 # Prepare the settings in the Escoria UI category
 func set_escoria_ui_settings():
-	escoria.register_setting(
-		"escoria/ui/default_dialog_type",
+	escoria.project_settings_manager.register_setting(
```

Comment:
I wonder if we should move all this into ESCProjectManager and just call escoria.project_settings_manager.initialize_settings()

Reply:
Up for discussion on that. :) I guess that depends on how fine-grained we want to get with breaking up settings (I think).